dev-test/config.yml

```diff
@@ -117,11 +117,11 @@ collections: # A list of collections the CMS should be able to edit
         display_fields: ['title', 'date']
         search_fields: ['title', 'body']
         value_field: 'title'
-      - { label: 'Title', name: 'title', widget: 'string' }
-      - { label: 'Boolean', name: 'boolean', widget: 'boolean', default: true }
+      - { label: 'Title', name: 'title', widget: 'string', prefix: 'This string:', suffix: 'is a title' }
```

Comment:
this suffix doesn't seem to make a lot of sense, is it a prefix ? is it a hint ? why would i want this ?

Reply:
Nope, it's just for testing and demonstration. But you are welcome to improve it.